modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/DartJaguarClientCodegen.java

```diff
@@ -152,6 +194,19 @@ public void processOpts() {
                     modelImports.add(underscore(modelImport));
                 }
             }
+
+            int index = 1;
+            for (CodegenProperty p : cm.vars) {
+                p.vendorExtensions.put("x-index", index);
+                String protoType = protoTypeMapping.get(p.baseType);
+                if(p.isListContainer) {
+                    String innerType = protoTypeMapping.get(p.mostInnerItems.baseType);
```

Comment:
this is not working yet, as here we have the dart type and not the openapi type :(

Reply:
This is now ok !